components/tabs/tabs.js

```diff
@@ -330,7 +333,8 @@ class Tabs extends LocalizeCoreElement(ArrowKeysMixin(SkeletonMixin(LitElement))
 			<div class="${classMap(panelContainerClasses)}"
 				@d2l-tab-panel-selected="${this._handlePanelSelected}"
 				@d2l-tab-panel-text-changed="${this._handlePanelTextChange}">
-				<slot @slotchange="${this._handlePanelsSlotChange}"></slot>
+				<slot @slotchange="${this._handleDefaultSlotChange}"></slot>
```

Comment:
If you want to make it extra clear for us later, we could name this `_handleLegacySlotChange` or something that makes it clear that this will no longer be supported. I don't want to suggest a lot of unnecessary change to your current PRs though. Up to you.

Reply:
I'd been debating the wording of using `Legacy` vs `Default`. `Legacy` suggests we plan on removing it, but in case the backport took a bit I didn't know if we'd want to leave that in there for a while.